### PR TITLE
Doc-427: Updated the sub-zones file for CENM 1.3, 1.4, and 1.5

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by jmacmahonr3 — CHANGES_REQUESTED** [review]
Hey 
The changes to the text look great. 
The only change request i'd like to see is that the signer needs to be within the compatibility zone. In the two sub zone diagrams

File: content/en/docs/cenm/1.3/sub-zones.md [hunk 1]
```diff
@@ -9,66 +9,88 @@ menu:
...
+This is an internal feature. Running a network with multiple subzones is not a supported configuration.
 
 {{< /note >}}
 
-## From a Node’s Perspective
+## Network from a Node’s Perspective
 
-From the perspective of a node a network is defined by the Identity Manager and Network Map Services it is configured
-to connect to. It has no comprehension of sub zones. It simply connects to the services configured within its
+From the perspective of a node a network is defined by the Identity Manager and Network Map services it is configured
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.3/sub-zones.md:25]:
```suggestion
From the perspective of a node, a network is defined by the Identity Manager and Network Map services it is configured
```

File: content/en/docs/cenm/1.3/sub-zones.md [hunk 2]
```diff
@@ -9,66 +9,88 @@ menu:
...
+{{<
+  figure
+	 src="resources/node-zone-view.png"
+	 zoom="resources/node-zone-view.png"
+   width=90%
+	 figcaption="Network from a Node's Perspective"
+	 alt="node zone view"
+>}}
+
+The node is unaware of other subzones, seeing only those nodes registered with the Network Map service it itself has
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.3/sub-zones.md:39]:
```suggestion
The node is unaware of other subzones - it sees only those nodes registered with the Network Map service that it has also
```

File: content/en/docs/cenm/1.3/sub-zones.md [hunk 3]
```diff
@@ -9,66 +9,88 @@ menu:
...
 registered with.
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.3/sub-zones.md:40]:
```suggestion
registered with itself.
```

Comment (taylorharvey-r3) [reply content/en/docs/cenm/1.3/sub-zones.md:40]: Tried to reword this sentence to make it a bit clearer.

File: content/en/docs/cenm/1.3/sub-zones.md [hunk 4]
```diff
@@ -9,66 +9,88 @@ menu:
...
+## Segregated Subzones
 
+The fundamental difference between a public subzone and a segregated one is that the operation of the notaries is
+deferred to a third party. The relationship between the zone operator and the notary operator is left to the discretion
+of the zone operator.
 
 {{< important >}}
-The relationship between the zone operator and the notary operator is left to the discretion
-of the zone operator. The important part from the perspective of the CENM is that the signed Node Info
+The important part from the perspective of the CENM is that the signed node info
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.3/sub-zones.md:81]:
```suggestion
The important part from the perspective of CENM is that the signed node info
```

File: content/en/docs/cenm/1.4/sub-zones.md [hunk 5]
```diff
@@ -9,66 +9,88 @@ menu:
...
+This is an internal feature. Running a network with multiple subzones is not a supported configuration.
 
 {{< /note >}}
 
-## From a Node’s Perspective
+## Network from a Node’s Perspective
 
-From the perspective of a node a network is defined by the Identity Manager and Network Map Services it is configured
-to connect to. It has no comprehension of sub zones. It simply connects to the services configured within its
+From the perspective of a node a network is defined by the Identity Manager and Network Map services it is configured
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.4/sub-zones.md:25]:
```suggestion
From the perspective of a node, a network is defined by the Identity Manager and Network Map services it is configured
```

File: content/en/docs/cenm/1.4/sub-zones.md [hunk 6]
```diff
@@ -9,66 +9,88 @@ menu:
...
+{{<
+  figure
+	 src="resources/node-zone-view.png"
+	 zoom="resources/node-zone-view.png"
+   width=90%
+	 figcaption="Network from a Node's Perspective"
+	 alt="node zone view"
+>}}
+
+The node is unaware of other subzones, seeing only those nodes registered with the Network Map service it itself has
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.4/sub-zones.md:39]:
```suggestion
The node is unaware of other subzones - it sees only those nodes registered with the Network Map service that it has also
```

File: content/en/docs/cenm/1.4/sub-zones.md [hunk 7]
```diff
@@ -9,66 +9,88 @@ menu:
...
 registered with.
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.4/sub-zones.md:40]:
```suggestion
registered with itself.
```

File: content/en/docs/cenm/1.4/sub-zones.md [hunk 8]
```diff
@@ -9,66 +9,88 @@ menu:
...
+## Segregated Subzones
 
+The fundamental difference between a public subzone and a segregated one is that the operation of the notaries is
+deferred to a third party. The relationship between the zone operator and the notary operator is left to the discretion
+of the zone operator.
 
 {{< important >}}
-The relationship between the zone operator and the notary operator is left to the discretion
-of the zone operator. The important part from the perspective of the CENM is that the signed Node Info
+The important part from the perspective of the CENM is that the signed node info
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.4/sub-zones.md:81]:
```suggestion
The important part from the perspective of CENM is that the signed node info
```

File: content/en/docs/cenm/1.5/sub-zones.md [hunk 9]
```diff
@@ -9,66 +9,88 @@ menu:
...
+This is an internal feature. Running a network with multiple subzones is not a supported configuration.
 
 {{< /note >}}
 
-## From a Node’s Perspective
+## Network from a Node’s Perspective
 
-From the perspective of a node a network is defined by the Identity Manager and Network Map Services it is configured
-to connect to. It has no comprehension of sub zones. It simply connects to the services configured within its
+From the perspective of a node a network is defined by the Identity Manager and Network Map services it is configured
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.5/sub-zones.md:25]:
```suggestion
From the perspective of a node, a network is defined by the Identity Manager and Network Map services it is configured
```

File: content/en/docs/cenm/1.5/sub-zones.md [hunk 10]
```diff
@@ -9,66 +9,88 @@ menu:
...
+{{<
+  figure
+	 src="resources/node-zone-view.png"
+	 zoom="resources/node-zone-view.png"
+   width=90%
+	 figcaption="Network from a Node's Perspective"
+	 alt="node zone view"
+>}}
+
+The node is unaware of other subzones, seeing only those nodes registered with the Network Map service it itself has
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.5/sub-zones.md:39]:
```suggestion
The node is unaware of other subzones - it sees only those nodes registered with the Network Map service that it has also
```

File: content/en/docs/cenm/1.5/sub-zones.md [hunk 11]
```diff
@@ -9,66 +9,88 @@ menu:
...
 registered with.
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.5/sub-zones.md:40]:
```suggestion
registered with itself.
```

File: content/en/docs/cenm/1.5/sub-zones.md [hunk 12]
```diff
@@ -9,66 +9,88 @@ menu:
...
+## Segregated Subzones
 
+The fundamental difference between a public subzone and a segregated one is that the operation of the notaries is
+deferred to a third party. The relationship between the zone operator and the notary operator is left to the discretion
+of the zone operator.
 
 {{< important >}}
-The relationship between the zone operator and the notary operator is left to the discretion
-of the zone operator. The important part from the perspective of the CENM is that the signed Node Info
+The important part from the perspective of the CENM is that the signed node info
```
Comment (taylorharvey-r3) [inline content/en/docs/cenm/1.5/sub-zones.md:81]:
```suggestion
The important part from the perspective of CENM is that the signed node info
```

**Review by taylorharvey-r3 — COMMENTED** [review]
Overall looks good, just a few comments that you can take or leave.

**Review by taylorharvey-r3 — APPROVED** [review]
LGTM

**Pull request merged**